pootle/apps/pootle_app/management/commands/clear_stats.py

```diff
@@ -12,11 +12,42 @@
 # This must be run before importing Django.
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
 
-from pootle_app.management.commands import PootleCommand
+from django.core.management.base import NoArgsCommand
 
+from django_redis import get_redis_connection
+from redis import WatchError
 
-class Command(PootleCommand):
-    help = "Flush stats cache."
+from pootle.core.mixins.treeitem import POOTLE_DIRTY_TREEITEMS
+from pootle.core.models import Revision
 
-    def handle_all_stores(self, translation_project, **options):
-        translation_project.clear_all_cache()
+
+class Command(NoArgsCommand):
+    help = """Flush stats cache and dirty tree item counter and
+    rq data (optional).
+    """
+
+    def handle_noargs(self, **options):
+        # Delete all stats cache data
+        r_con = get_redis_connection('stats')
+        r_con.flushdb()
+
+        # Clear all dirty tree items
+        r_con = get_redis_connection('redis')
+        r_con.delete(POOTLE_DIRTY_TREEITEMS)
+
+        clear_rq = options.pop("clear-rq", False)
```

Comment:
What I was trying to say is that `NoArgsCommand` is not providing this option, so you have to register it like in https://github.com/translate/pootle/blob/master/pootle/apps/pootle_app/management/commands/changed_languages.py#L29-L37